src/js/App/FeatureFlags/FeatureFlagsProvider.tsx

```diff
@@ -18,14 +21,38 @@ export const UNLEASH_ERROR_KEY = 'chrome:feature-flags:error';
  */
 localStorage.setItem(UNLEASH_ERROR_KEY, 'false');
 
-export const unleashClient = new UnleashClient(config);
+export let unleashClient: UnleashClient;
 export const getFeatureFlagsError = () => localStorage.getItem(UNLEASH_ERROR_KEY) === 'true';
 
-unleashClient.on('error', () => {
-  localStorage.setItem(UNLEASH_ERROR_KEY, 'true');
-});
-
-const FeatureFlagsProvider: React.FC = ({ children }) => <UnleasFlagProvider unleashClient={unleashClient}>{children}</UnleasFlagProvider>;
+const FeatureFlagsProvider: React.FC = ({ children }) => {
+  const user = useSelector<ReduxState, ChromeUser | undefined>((state) => state.chrome.user);
+  const unleashClientInternal = useRef<UnleashClient>();
+
+  // create the unleash client only after the user object is avaiable
+  useEffect(() => {
+    if (user && !unleashClientInternal.current) {
+      unleashClientInternal.current = new UnleashClient({
+        ...config,
+        context: {
+          properties: {
+            account_number: user.identity.account_number,
+          },
+        },
+      });
+      unleashClient = unleashClientInternal.current;
+      unleashClient.on('error', () => {
+        localStorage.setItem(UNLEASH_ERROR_KEY, 'true');
+      });
+    }
+  }, [user]);
+
+  // fallback to render the chrome withouth the feature flags if the provider is not initialized properly
+  if (getFeatureFlagsError() || !unleashClientInternal.current) {
+    return <>{children}</>;
```

Comment:
I preder the full component name
```suggestion
    return <>{children}</>;
```
```suggestion
    return <React.Fragment>{children}</React.Fragment>;
```